test/test_retriever.py

```diff
@@ -32,16 +30,19 @@
                                        shortname='test')
 test_engine.opts = {'database_name': '{db}_abc'}
 HOMEDIR = os.path.expanduser('~')
+file_location = os.path.dirname(os.path.realpath(__file__))
+site_dir = os.path.abspath(os.path.join(file_location, os.pardir))
 
 
 def setup_module():
-    """"change directory to test directory"""
-    os.chdir(os.path.dirname(os.path.abspath(__file__)))
+    """"Make sure you are in the main local retriever directory"""
+    os.chdir(site_dir)
+    os.system('cp -r {0} {1}'.format(os.path.join(site_dir, "test/raw_data"), site_dir))
```

Comment:
Do we use the raw data in this test module? Since this is all unit tests I'd expect that we don't, in which case we could drop this line here.